### PR TITLE
created movie and tvSeries page. Added profile template page

### PR DESCRIPTION
**Pull request by gurnavdeep**
Can you please guys check it.
Just have implemented lists of movies and tvSeries, routes haven't been added yet.
Thanks

### CODE REVIEW

File: src/Controller/MovieController.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class MovieController extends AbstractController
+{
+    /**
+     * @Route("/movie", name="movie")
+     */
+    public function index()
+    {
+        $movies = $this->getDoctrine()->getRepository('App:Movie')->findAll();
+        //dump($movies);
```
Comment (RedHotMan) [inline src/Controller/MovieController.php]: You should remove those `dump` code. Useful in development mode only

File: src/Controller/TvController.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+class TvController extends AbstractController
+{
+    /**
+     * @Route("/tv", name="tv")
+     */
+    public function index()
+    {
+        $serials = $this->getDoctrine()->getRepository('App:Tv')->findAll();
+        //dump($movies);
```
Comment (RedHotMan) [inline src/Controller/TvController.php]: Same as earlier

File: templates/index.html.twig [hunk 3]
```diff
@@ -67,6 +67,14 @@
...
                                 {% endif %}
+
+                                {% if app.user %}
+                                    <span class="m-2 d-inline-block invisible" data-toggle="appear" data-class="animated fadeInUp" data-timeout="600">
+                                        <a class="btn btn-success px-4 py-2" data-toggle="click-ripple" href='{{ path('fos_user_profile_show') }}'>
+                                            <i class="fa fa-fw fa-shopping-cart mr-1"></i> Profile
+                                        </a>
+                                    </span>
+                                {% endif %}
                             </div>
```
Comment (RedHotMan) [inline templates/index.html.twig]: You could add two buttons to access to movie and tv list even if use is not connected

File: .gitignore [hunk 4]
```diff
@@ -5,6 +5,8 @@
 /var/
 .idea/
 ###< symfony/framework-bundle ###
+.DS_Store
+.OneUI-Source
```
Comment (CNadjim) [inline .gitignore]: c'est pas plutôt /OneUI-Source ?

File: templates/movie/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,213 @@
+
```
Comment (CNadjim) [inline templates/movie/index.html.twig]: On devrait pas faire un template générique, et dans la balise content tu mets le contenu de la route, sinon yaura grave du code redondant

File: templates/tv/index.html.twig [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+            <!-- Page Content -->
+            <div class="content">
+
+                <div class="row">
+                    {% for serial in serials %}
+                        <div class="col-md-4">
+                            <div class="block">
+                                <div class="block-header">
+                                    <h3 class="block-title">{{ serial.name }} </h3> <small>Original Language : {{ serial.originalLanguage }}</small>
```
Comment (RedHotMan) [inline templates/tv/index.html.twig]: I think we could make something better with the display here. 
Maybe fix the max length and width, or something like that

File: templates/movie/index.html.twig (hunk 5 above)
Comment (RedHotMan) [reply templates/movie/index.html.twig]: Je suis d'accord

File: .gitignore (hunk 4 above)
Comment (gurnavdeep) [reply .gitignore]: Exact c'est bien /OneUI-Source

File: src/Controller/MovieController.php (hunk 1 above)
Comment (gurnavdeep) [reply src/Controller/MovieController.php]: Yes in the end it will be deleted

File: templates/index.html.twig (hunk 3 above)
Comment (gurnavdeep) [reply templates/index.html.twig]: Yes Sir. Like i said in the pull request, routes haven't been implemented

File: templates/tv/index.html.twig (hunk 6 above)
Comment (gurnavdeep) [reply templates/tv/index.html.twig]: Not the final design. It will be reworked later.

**Pull request merged**